src/txmempool.cpp

```diff
@@ -753,9 +777,15 @@ void CTxMemPool::removeConflicts(const CScCertificate &cert,std::list<CTransacti
         }
     }
 
-    for (auto& conflictingCert : vLowerQualCerts)
-    {
+    for (const auto& conflictingCert : vLowerQualCerts)
         remove(conflictingCert, removedTxs, removedCerts, true);
+
+    //remove all btrs for scIds mentioned by the certificate. These btrs would target outdated scFees
```

Comment:
As I remember, the reason will be mostly in different CertificateDataCumulativeHash. `scFees` can be still valid.

Reply:
Agree